CompuCell3D/core/CompuCell3D/plugins/ImplicitMotility/ImplicitMotilityPlugin.cpp

```diff
@@ -189,7 +193,14 @@ double ImplicitMotilityPlugin::changeEnergyByCellType(const Point3D &pt, const C
 
 		Coordinates3D<double> newCOMBeforeFlip(newCell->xCM / newCell->volume, newCell->yCM / newCell->volume, newCell->zCM / newCell->volume);
 		Coordinates3D<double> distVector = distanceVectorCoordinatesInvariant(newCOMAfterFlip, newCOMBeforeFlip, fieldDim);
-		//externalPotentialParamVector[newCell->type]
+		
+		double norm = std::sqrt(distVector.X()*distVector.X() + distVector.Y()*distVector.Y() + distVector.Z()*distVector.Z());
```

Comment:
I would make  function double norm(distVector). This way the code woudl be less verbose. Not essential to do it now though